tests/get_coverage.sh

```diff
@@ -32,10 +31,12 @@ call_ansible() {
 }
 
 remote_coverage_dir="$(mktemp -d /tmp/remote_coverage-XXXXXX)"
-trap "rm -rf '${remote_coverage_dir}'" EXIT
+trap 'rm -rf "${remote_coverage_dir}"' EXIT
```

Comment:
Is this change correct? AFAIU, this will make `$remote_coverage_dir` to be evaluated when the script exits but it should use the value it has at the time the trap is configured.

Reply:
shellcheck assumes that the correct thing to do is to use single quotes around the entire `trap` command, and only expand it, including any variables used by the `trap` command, at `trap` evaulation time.  https://github.com/koalaman/shellcheck/wiki/SC2064 - but we know what we're doing here, and we want the variable expanded at the point where the `trap` is declared.  So we follow the directions:
```

Exceptions

If you don't care that the trap code is expanded early because the commands/variables won't change during execution of the script, or because you want to use the current and not the future values, then you can ignore this message.
```
so we will tell shellcheck to be quiet